components/Testimonial.jsx

```diff
@@ -0,0 +1,48 @@
+import styled from "styled-components";
+import { ChatRightQuoteFill } from "@styled-icons/bootstrap/ChatRightQuoteFill";
+import { media } from "../utils";
+import { Text } from "./Text";
+
+export const Testimonial = ({ testimonial, author, ...props }) => {
+  return (
+    <Wrapper {...props}>
+      <Logo />
+      <SText>{testimonial}</SText>
+      <SText bold="true">{author}</SText>
+    </Wrapper>
+  );
+};
+
+const Wrapper = styled.div`
+  width: 60%;
+  padding: 5%;
```

Comment:
I'd decrease the padding on larger screens because there's a lot of padding/text is small. More padding on <500 is nice though
![image](https://user-images.githubusercontent.com/65516245/129087927-a5bbbf04-36eb-44e5-b090-ab11e3f9b32b.png)